MapleServer2/Types/RegionSkillHandler.cs

```diff
@@ -156,7 +164,7 @@ private static List<CoordF> GetEffectCoords(SkillCast skillCast, int mapId, int
 
             // Since this is the block below, add 150 units to the Z coord so the effect is above the block
             offSetCoord = blockBelow.Coord.ToFloat();
-            offSetCoord.Z += Block.BLOCK_SIZE;
+            offSetCoord.Z += Block.BLOCK_SIZE + 1;
```

Comment:
what is suppose to mean this `+1` ?

Reply:
It was to line up the values more with what shock whip was sending to see if it would have an effect on making it work

Reply:
Only +1 is needed?